client/src/pages/ManageBookings/ManageBookings.tsx

```diff
@@ -0,0 +1,141 @@
+import { useState } from 'react';
+import PageContainer from '../../components/PageContainer/PageContainer';
+import clsx from 'clsx';
+import { Box, Grid, Typography, Avatar } from '@mui/material';
+import useStyles from './useStyles';
+import CardWrapper from '../../components/CardWrapper/CardWrapper';
+import SettingsIcon from '@mui/icons-material/Settings';
+import { NavLink } from 'react-router-dom';
+import dogPicture from '../../images/landing/hero.jpg';
+
+import MyCalendar from '../../components/MyCalendar/MyCalendar';
+
+const RequestInfo: React.FC<{
+  date?: string | Date;
+  avatar?: any;
+  name?: string;
+  status?: string;
+  size?: any;
+}> = ({ date, avatar, name, status, size }) => {
+  const classes = useStyles();
+
+  return (
+    <Box className={clsx(classes.requestInfo, size === 'large' && classes.requestInfoLarge)}>
+      <Grid xs={9} item>
+        <Typography variant={size === 'large' ? 'h6' : 'body1'} gutterBottom>
+          {date}
+        </Typography>
+        <Box className={classes.nameAvatar}>
+          <Avatar
+            alt="Profile Image"
+            src={avatar}
+            sx={size === 'large' ? { width: 50, height: 50 } : { width: 40, height: 40 }}
+          />
+          <Typography variant={size === 'large' ? 'h6' : 'body1'}>{name}</Typography>
+        </Box>
+      </Grid>
+      <Grid xs={3} item className={classes.statusAndIcon}>
+        <Typography variant="caption" className={classes.status}>
+          {size === 'large' ? '' : status}
+        </Typography>
+        <NavLink to="" className={clsx(classes.settingsIcon, size === 'large' && classes.settingsIconLarge)}>
+          <SettingsIcon color="disabled" fontSize="small" />
+        </NavLink>
+      </Grid>
+    </Box>
+  );
+};
+
+export default function ManageBookings(): JSX.Element {
+  const classes = useStyles();
+  const [dateValue, setDateValue] = useState<Date>(new Date(new Date().getFullYear(), new Date().getMonth(), 20));
+
+  return (
+    <PageContainer>
+      <Grid sx={{ width: '87%', margin: '0 auto' }} spacing={2} container>
+        <Grid item xs={12} md={6} order={{ xs: 2, md: 1 }} className={classes.leftContainer}>
+          <Box className={classes.leftWrapper}>
+            <CardWrapper>
+              <Box className={classes.bookingContent}>
+                <Typography className={classes.title} variant="caption" gutterBottom>
+                  Your Next booking:
+                </Typography>
+                <RequestInfo
+                  key="123"
```

Comment:
Rather than passing multiple props, it would be easier to pass the entire booking object to a request card